attributes/ckan.rb

```diff
@@ -28,7 +28,7 @@
     ['postgresql94-devel', 'postgresql-devel'],
     ['python27-devel', 'python3-devel'],
     ['python27-pip', 'python3-pip'],
-    ['mod24_wsgi-python27', 'mod_wsgi']
+    ['mod24_wsgi-python27', 'python3-mod_wsgi']
```

Comment:
hmm, guess you missed it or it was in-decisive

Reply:
`mod_wsgi` exists on Amazon Linux 2, but it turns out to be compiled for Python 2, and cannot work with Python 3. Which, incidentally, *is not mentioned* in the `yum info` for the package. :|